debian/stretch/foreman/control

```diff
@@ -6,7 +6,7 @@ Standards-Version: 3.9.1
 Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.3), ruby2.3-dev, git | git-core,
                zlib1g-dev, default-libmysqlclient-dev, libpq-dev, pkg-config, libvirt-dev,
                libsqlite3-dev, build-essential, curl, nodejs (>= 8.9),
-               python
+               python, dh-systemd (>= 1.5)
```

Comment:
`dh_systemd` is part of `debhelper` since 9.20160709 (stretch has 10.2.5), so you can safely avoid that dependency if you bump the `debhelper` one accodingly.

Reply:
Jessie only has 9.20150101, so we'd need jessie-backports in our buildroots for that to work. But if that's the case, :+1: on doing `debhelper (>= 9.20160709)`